### PR TITLE
#2791 - Add missing env variable IS_FULLTIME_ALLOWED to queue consumers

### PR DESCRIPTION
**Pull request by dheepak-aot**
Adding Missing env variable IS_FULLTIME_ALLOWED from the task #2524

### CODE REVIEW

File: devops/openshift/queue-consumers-deploy.yml [hunk 1]
```diff
@@ -267,18 +269,18 @@ objects:
...
+            name: cpu
+            target:
+              type: Utilization
+              averageUtilization: 80
+        - type: Resource
+          resource:
+            name: memory
+            target:
+              type: AverageValue
+              averageValue: "230M"
```
Comment (andrepestana-aot) [inline devops/openshift/queue-consumers-deploy.yml:283]: Why was it removed?

Comment (dheepak-aot) [reply devops/openshift/queue-consumers-deploy.yml:283]: Nothing was removed. It was just re-formatted. 
Btw. I use the YML plugin as we all discussed.

image

image

Comment (andrepestana-aot) [reply devops/openshift/queue-consumers-deploy.yml:283]: yes, I quickly checked and thought it was removed. OK. Thank you!

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/docker-compose.yml [hunk 2]
```diff
@@ -150,6 +150,7 @@ services:
       - ESDC_ENVIRONMENT_CODE=${ESDC_ENVIRONMENT_CODE}
       - APPLICATION_ARCHIVE_DAYS=${APPLICATION_ARCHIVE_DAYS}
       - QUEUE_CONSUMERS_PORT=${QUEUE_CONSUMERS_PORT}
+      - IS_FULLTIME_ALLOWED=${IS_FULLTIME_ALLOWED}
```
Comment (sh16011993) [inline sources/docker-compose.yml:153]: Why do we need it in the queue consumers? I am sorry if I missed it from some earlier discussion in the meetings. 

We are getting the value for `IS_FULLTIME_ALLOWED` from the api as below:

image

Please let me know if I am missing something.

**Review by sh16011993 — COMMENTED** [review]
Just one info comment but everything else looks good 👍

File: sources/docker-compose.yml (hunk 2 above)
Comment (dheepak-aot) [reply sources/docker-compose.yml:153]: The portion of code that you shared in on queue consumers NOT api and queue consumers is missing the ENV variable.

Comment (sh16011993) [reply sources/docker-compose.yml:153]: Thanks for the clarification on the call @dheepak-aot

**Review by sh16011993 — APPROVED** [review]
Nice work 👍 LGTM @dheepak-aot

**Review by lewischen-aot — APPROVED** [review]
LGTM. Nice work @dheepak-aot 👍

**Pull request merged**